README.md

```diff
@@ -39,55 +73,206 @@ tools (such as `ceph-installer` and `gdeploy`):
+  tendrl components and etcd instance).
+
+
+## How do I install Tendrl with tendrl-ansbile?
+
+1)  Install tendrl-ansible:
+
+    ```
+    # yum install tendrl-ansible
+    ```
+
+    See section "How to get tendrl-ansible?" in this README file for more
+    details.
+
+2)  Create [Ansible inventory
+    file](https://docs.ansible.com/ansible/latest/intro_inventory.html) with
+    groups for `tendrl-server` and `gluster-servers`. Here is an example of
+    inventory file for 4 node cluster with Gluster:
+
+    ```
+    [gluster-servers]
+    gl1.example.com
+    gl2.example.com
+    gl3.example.com
+    gl4.example.com
+
+    [tendrl-server]
+    tendrl.example.com
+    ```
+
+3)  Add mandatory ansible variables into the inventory file you created in the
+    previous step.
+
+    This includes:
+
+    * `etcd_ip_address` configures where etcd instance is listening
+    * `etcd_fqdn` configure tendrl components to be able to connect to etcd
+      instance
+    * `graphite_fqdn` configures tendrl components (this value doesn't
+       reconfigure graphite itself!) to be able to connect to graphite instance
+
+    For simple example cluster from previous step, assuming there is only
+    single network interface on all machines, the code you need to add into
+    the inventory file would look like:
+
+    ```
+    [all:vars]
+    etcd_ip_address=192.0.2.1
+    etcd_fqdn=tendrl.example.com
+    graphite_fqdn=tendrl.example.com
+    ```
+
+    Where `192.0.2.1` is ip address of tendrl server, `tendrl.example.com` is
+    a hostname of tendrl server and `tendrl.example.com` hostname is translated
+    to `192.0.2.1` ip address.
+
+    See full description in README file of tendrl-server role and pay attention
+    to the values you specify there when you use multiple network interfaces
+    on the machines.
+
+    Note: you can define these variables anywhere else you like (eg. in
+    variable files or from command line directly), but including them into the
+    inventory provides you with a single file with full description of
+    tendrl-ansible setup for future reference (eg. reruning tendrl-ansible
+    later when you need to expand cluster or make sure the configuration still
+    holds).
+
+4)  Add optional ansible variables into the invenory file.
+
+    Based on Tendrl documentation and description in README files of
+    tendrl-ansible roles, specify values for variables you like to tweak.
+
+    This is important because some features tendrl-ansible can help you
+    with are disabled by default as they require additional user input.
+
+    This includes etcd tls client authentication and tendrl notifier
+    configuration for snmp or smtp. Other features such as firewalld setup
+    for Tendrl can be disabled if needed.
+
+5)  If you use tendrl-ansible from rpm package, copy `site.yml` playbook into
+    local directory (where you already store the inventory file):
```

Comment:
I think, that *working* directory would be slightly better than *local* directory.

Reply:
ok, fixed in a2646c06cc54b48975468e76761fcd4aba23c454